packages/search/addon/components/navi-search-result/base.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2020, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ *
+ * Base search result component
+ */
+
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+
+/**
+ * @interface Args
+ */
+interface Args {
+  data: Array<Object>;
```

Comment:
Can we define a better shape here instead of just object? Or maybe use TODO type

Reply:
This `Object` would be what each provider would return, so say the Asset provider would return a `report` or `dashboard` object or the Definition provider would return records from the keg. I can switch it to TODO but I feel it'd be hard to model exactly.